src/Connection/ConnectionInterface.php

```diff
@@ -69,6 +69,15 @@ public function getServerVersion(): string;
      */
     public function getTableSchema(string $name, $refresh = false): ?TableSchema;
 
+    /**
+     * Whether to enable auto recogintion of read queries and use slave (if enabled) for execute
+     *
+     * @return bool:
+     *  true -> (default) use slave for read queries, master for other
+     *  false -> always use master connection (for reads and writes). Slave still can be used via $this->getSlave()
+     */
+    public function isAutoSlaveForReadQueriesEnabled(): bool;
+
     /**
```

Comment:
/**
     * Whether to enable auto recogintion of read queries and use slave (if enabled) for execute.
     *
     * @return bool For default `true` use slave for read queries, `false` use master master connection (for reads and
     * writes). Slave still can be used via $this->getSlave().
     */
    public function isAutoSlaveForReadQueriesEnabled(): **bool;
```